src/components/PlayerDash.jsx

```diff
@@ -15,38 +17,47 @@ export const PlayerDash = React.createClass({
   getInitialState() {
     return {
       players: [],
+      graphdata: [],
       loaded: false
     }
   },
 
   componentWillMount() {
+    this.firebase = new FirebaseLib();
     // We are likely to want to read all player data e.g. top adversary, last 10 matches etc
     // So leaving this here for now. Woould be nicer to have a store where this data is kept without requesting again
-    let fbPath = [conf.firebaseUrl, 'players'].join('/');
-    this.fireBase = new Firebase(fbPath);
     this.loadData(); // should update to bindAsObject/Array
   },
 
   render() {
```

Comment:
Wicked man. Yeah agree with 'the' statement. 

Don't worry comments are wicked as.